mlos_bench/mlos_bench/tunables/tunable.py

```diff
@@ -92,8 +92,8 @@ def _sanity_check(self) -> None:
                 raise ValueError(f"Range must be None for the categorical type tunable {self}")
             if len(set(self._values)) != len(self._values):
                 raise ValueError(f"Values must be unique for the categorical type tunable {self}")
-            if self._special is not None:
-                raise ValueError(f"Special values must be None for the categorical type tunable {self}")
+            if self._special:
```

Comment:
I think this may fail for `False` special values.
Not sure why we'd have those but better to support and avoid weird hard to debug conditions for users I think.
Or throw a warning in that case?

Reply:
nope - `_special` is always a list, and if there are no special values, it is equal to `[]`. If `False` is a special value, `self._special` will contain `[False]` (i.e., a list of 1 value)